### ISSUE TITLE
Use current .git as default

### ISSUE THREAD

**Issue by koraa** [donnemartin/gitsome#60]
When using commands that require a repo/context, the current git remotes should be used.
E.g. when I type `gh issues` in the local git directory of a github project, it should list 
all the issues of that project. Same with `gh repo` and similar commands.

**Comment by donnemartin** [comment]
Thanks for the suggestion!

**Comment by codeHatcher** [comment]
agree, the current command line utility i'm using has this and it is much much better and easier to use. right now when I do gh issues I get 100 and only about 5 are for my current repo
